study.md

```diff
@@ -40,14 +41,16 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+It changes the method being called permenantly  otherwise it would create a copy that could be manipulated
```

Comment:
`!` signifies that the method will modify the object that the method was invoked upon rather than return a copy of it.